src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js

```diff
@@ -88,7 +88,9 @@ const DetailHeader = ({ incident, baseUrl, onPatchIncident }) => {
     },
   };
 
-  const referrer = location.referrer || `${baseUrl}/incidents`;
+  const mapOverviewLocation = '/manage/incidents/kaart';
```

Comment:
There is a named export for this route in `routes.js` in the `IncidentManagement` module.

Reply:
Awesome! Thanks!